src/sdk/lib/StorageManager.js

```diff
@@ -108,3 +108,16 @@ export class LocalStorageManager extends BaseStorageManager
         super( storageName, window.localStorage );
     }
 }
+
+export const getStorageClass = (opts) => {
+    // Only cookie support domain based storage
+    if (opts.storageType === 'cookie' || opts.cookieDomain) {
```

Comment:
hey actually I think it's better to check the && for the conditions. 
both have to be present in order to choose cookie.